src/main/java/school/hei/haapi/repository/FeeRepository.java

```diff
@@ -25,4 +25,7 @@ List<Fee> getFeesByStudentIdAndStatus(String studentId,
       + "and f.remainingAmount > 0 "
       + "and f.dueDatetime < current_date")
   List<Fee> getUnpaidFees();
+
+  @Query("select f from Fee f where f.status = 'LATE'")
```

Comment:
Is there already an index on the "status" column in database ?

Reply:
Yes there already is an index on the "status" column